docs/_docs/2_2_writing_data.md

```diff
@@ -210,6 +210,8 @@ Sample config files for table wise overridden properties can be found under `hud
   --op BULK_INSERT
 ```
 
+For detailed information on how to configure and use `HoodieMultiTableDeltaStreamer`, please refer [blog section](/blog/ingest-multiple-tables-using-hudi).
```

Comment:
I don't know, just confirming. do you know this link actually works.

Reply:
yeah, it works :) I have tested.